components/shared/searchFilters/helper.ts

```diff
@@ -0,0 +1,49 @@
+import { FacetField, SearchFilters } from "@/lib/graphql/generated/fbi/graphql"
+import { AppRouterInstance } from "next/dist/shared/lib/app-router-context.shared-runtime"
+
+export const facetDefinitions = [
+  "materialTypesGeneral",
+  "mainLanguages",
+  "age",
+  "lix",
+  "subjects",
+] as FacetField[]
+
+export const mapFacetsToFilters = {
+  materialTypesGeneral: "materialTypesGeneral",
+  mainLanguages: "mainLanguages",
+  age: "age",
+  lix: "lixRange",
+  subjects: "subjects",
+} as Record<FacetField, keyof SearchFilters>
+
+export const mapFilterNameToTranslation: Partial<Record<keyof typeof mapFacetsToFilters, string>> =
+  {
+    materialTypesGeneral: "Type",
+    mainLanguages: "Sprog",
+    age: "Alder",
+    lix: "Lix",
+    subjects: "Emne",
+  }
+
+export const toggleFilter = (filterName: string, value: string, router: AppRouterInstance) => {
+  const searchParams = new URLSearchParams(window.location.search)
```

Comment:
Although this is fine, I think it should be possible to get the query string from the `router`object.

Reply:
Did you follow up on this? You did not reply :)

Reply:
<img width="340" alt="image" src="https://github.com/user-attachments/assets/60f34030-8daa-43cf-8841-da59ae4e2b0a">
It doesn't seem like that's an option. Is it a big deal though?

Reply:
Nope :)